scripts/reindex.py

```diff
@@ -50,16 +50,17 @@
                     default=defaults.num_workers,
                     type=int,
                     help='The number of workers that will be sending bundles to the indexer concurrently')
-parser.add_argument('--partition-prefix-length',
-                    metavar='NUM',
-                    default=0,
-                    type=int,
-                    help='The length of the bundle UUID prefix by which to partition the set of bundles matching the '
-                         'query. Each query partition is processed independently and remotely by the indexer lambda. '
-                         'The lambda queries the repository and queues a notification for each matching bundle. If 0 '
-                         '(the default) no partitioning occurs, the repository is queried locally and the indexer '
-                         'notification endpoint is invoked for each bundle individually and concurrently using worker'
-                         'threads. This is magnitudes slower than remote i.e., partitioned indexing.')
+parser.add_argument('--remote',
+                    dest='remote',
+                    default=False,
+                    action='store_true',
+                    help='Perform the reindexing remotely. The partition prefix length for the catalog source is used '
+                         'to form queries that are processed independently and remotely by the indexer lambda. The '
+                         'lambda queries the repository and queues a notification for each matching subgraph. If the '
+                         'flag is set false (the default) no partitioning occurs, the repository is queried locally '
+                         'and the indexer notification endpoint is invoked for each bundle individually and '
+                         'concurrently using worker threads. This is magnitudes slower than remote i.e., partitioned'
+                         ' indexing.')
```

Comment:
```suggestion
                    help='Perform the reindexing remotely. The set of subgraphs matching the query is partitioned using the ' 
                             'partition prefix length(s) configured for the catalog sources(s). Each query partition is processed '
                             'independently and remotely by the indexer lambda. The lambda queries the repository and '
                             'queues a notification for each matching subgraph. If this flag is not set, no partitioning occurs, '
                             'the repository is queried locally, and the indexer notification endpoint is invoked for each bundle '
                             'individually and concurrently using worker threads. This is magnitudes slower than remote (i.e. '
                             partitioned) indexing.')
```
```

Reply:
I've re-written this to try and preserve more of the original language which I think was clearer. I also made the following specific changes:
 - Replaced a lingering reference to "bundle" with subgraph".
 - Pluralized references to catalog sources since there can be more than one per catalog.
 - Added oxford comma and changed another comma I felt was awkward to parentheses. These changes are admittedly a bit subjective.
 - Moved leading whitespace to the previous line.

Reply:
Good thing you added this paragraph, it was an oversight on my part not to point out that you initially omitted it.

Reply:
Noticed it because of the small diff by moving `--remote` to where `--partition-prefix-length` was defined. :)